crates/pink/pink-extension/Cargo.toml

```diff
@@ -22,7 +22,7 @@ log = "0.4.17"
 insta = "1.7.2"
 
 [features]
-default = ["std", "runtime_utils"]
+default = ["std", "runtime_utils", "ink-as-dependency"]
```

Comment:
Why do we enable `ink-as-dependency` in the extension crate? Is it because of `trait_definition`?

Reply:
> Is it because of trait_definition?

Yeah